enclave/src/lib.rs

```diff
@@ -216,6 +216,9 @@ fn create_extrinsics(
     let signer = ed25519::unseal_pair()?;
     debug!("Restored ECC pubkey: {:?}", signer.public());
 
+    let mutex = nonce_handler::load_nonce_storage()?;
+    let mut nonce_storage: SgxMutexGuard<NonceHandler> = mutex.lock().unwrap();
```

Comment:
We risk running into a panic here?

Reply:
Yes, we do. I've created an issue because this solution is not really acceptable. https://github.com/Polkadex-Substrate/polkadexTEE-worker/issues/156